src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1192,12 +1194,31 @@
                AdditionalLibPackageExcludes="@(AdditionalLibPackageExcludes)"
                AdditionalSymbolPackageExcludes="@(AdditionalSymbolPackageExcludes)"
                Condition="'$(_SkipCreatePackage)' != 'true'"/>
+        
     <!-- Create a marker that records the path to the generated package -->
     <WriteLinesToFile Lines="$(PackageOutputPath)$(Id).$(PackageVersion).nupkg;$(SymbolPackageOutputPath)$(Id).$(PackageVersion).symbols.nupkg"
                       File="$(NuSpecPath).pkgpath"
                       Overwrite="true"
                       Condition="'$(_SkipCreatePackage)' != 'true'"/>
   </Target>
+  
+  <!-- Installs locally built package into a packageinstalldir for consuming -->
+  <!-- Overwrites previous installs, supports only one version installed -->
+  <Target Name="InstallLocallyBuiltPackages"
+    Condition="$(SkipInstallLocallyBuiltPackages)!='true'"
+    AfterTargets="CreatePackage">
+    
+    <!-- Force overwrite install directory of package to allow only one version installed -->
+    <RemoveDir Directories="$(PackageInstallPath)\$(Id)" />
+    
+    <Error Text="Package not installed as it was not found at $(PackageOutputPath)$(Id).$(PackageVersion).nupkg" 
```

Comment:
Do you want to do this check before you remove the previous directory.  Even though they're pointing to different paths, I'd expect to fail sooner rather than remove the directory and then fail.